Assets/Shopify/UIToolkit/Editor/ShopCredentialsVerifier.cs

```diff
@@ -34,12 +34,11 @@ public class ShopCredentialsView : IShopCredentialsView {
             DrawMessageBox();
             DrawActionButton();
             EditorGUILayout.Separator();
-            serializedObject.ApplyModifiedProperties();
         }
 
         private void DrawCredentialsForm(SerializedObject serializedObject) {
-            EditorGUILayout.PropertyField(serializedObject.FindProperty("_shopDomain"));
-            EditorGUILayout.PropertyField(serializedObject.FindProperty("_accessToken"));
+            EditorGUILayout.PropertyField(serializedObject.FindProperty("Credentials._Domain"));
+            EditorGUILayout.PropertyField(serializedObject.FindProperty("Credentials._AccessToken"));
```

Comment:
You can Keypath into other objects which is cool????????? Have to use the field name instead of the property here though because object serialization.

Reply:
Cool! I had no idea. So you're able to pull in a serialized private data member out of an object?

Reply:
Yea i think it has to do with the fact that everything is just reduced to key value pairs so the "private/public" accessors stop meaning anything once you are in serializedland